contributing.md

```diff
@@ -0,0 +1,71 @@
+# PWA-WP Contributing Guide
+
+Thanks for taking the time to contribute!
+
+To start, clone this repository into your WordPress install being used for development:
+
+```bash
+cd wp-content/plugins && git clone git@github.com:xwp/pwa-wp.git -wa
+```
+
+Lastly, to get the plugin running in your WordPress install, run `composer install` and then activate the plugin via the WordPress dashboard or `wp plugin activate pwa`.
+
+To install the `pre-commit` hook, do `bash dev-lib/install-pre-commit-hook.sh`.
+
+Note that pull requests will be checked against [WordPress-Coding-Standards](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards) with PHPCS, and for JavaScript linting is done with ESLint and (for now) JSCS and JSHint.
+
+## Creating a Plugin Build
+
+To create a build of the plugin for installing in WordPress as a ZIP package, do:
+
+```bash
+composer install # (if you haven't done so yet)
+npm install # (if you haven't done so yet)
+npm run build
+```
+
+This will create an `pwa.zip` in the plugin directory which you can install. The contents of this ZIP are also located in the `build` directory which you can `rsync` somewhere as well.
+
+To create a build of the plugin as it will be deployed to WordPress.org, run:
+
+```bash
+npm run build-release
+```
+
+Note that this will currently take much longer than a regular build because it generates the files required for translation. You also must have WP-CLI installed with the [`i18n-command` package](https://github.com/wp-cli/i18n-command).
```

Comment:
Let’s also link out to WP-CLI; I’d suggest changing the grammar here to “To run this command you must also have installed both [WP-CLI](#) and the [`i18n-command` package](#)`

Reply:
Good changes. But I copied this bit from the AMP plugin and it isn't relevant used in the PWA plugin (not yet anyway). So I'll just delete the paragraph for now, but I'll apply your patch to AMP.